portfolio/src/main/webapp/map.js

```diff
@@ -1,10 +1,47 @@
 /** Creates a map and adds it to the page. */
-const LATITUDE = 37.422;
-const LONGITUDE = -122.084;
-const ZOOM_FACTOR = 16;
+const ZOOM_FACTOR = 10;
+const LOCATIONS = ('[{"latitude":"39.750520", "longitude":"-105.222603", "title":"Colorado School of Mines", "description":"My college."}, \
+                     {"latitude":"40.151216", "longitude":"-105.166999", "title":"Silver Creek High School", "description":"My high school."}, \
+                     {"latitude":"40.022038", "longitude":"-105.254843", "title":"Google Boulder", "description":"Where I am working this summer! (virtually)"}]');
 
-function createMap() {
+function createMapAndMarkers() {
+    locations = JSON.parse(LOCATIONS);
+    centered_location = calculateCenterCoordinate(locations);
```

Comment:
centeredLocation I think "_" only apply to final variables that can be used as constant I might be wrong